Mail2Bug/Email/IIncomingEmailMessage.cs

```diff
@@ -15,6 +15,7 @@ public interface IIncomingEmailMessage
         string Subject { get; }
         string RawBody { get; }
         string PlainTextBody { get; }
+        string ConversationGuid { get; }
```

Comment:
Please add documentation of what the difference is between ConversationGuid vs ConversationIndex. Similar documentation should be added for the configuration knob, so people have a better idea of what they want to use